Documentation/upgrade.md

```diff
@@ -14,14 +14,17 @@ The goal is to provide prescriptive guidance and knowledge on how to upgrade a l
 We welcome feedback and opening issues!
 
 ## Supported Versions
-The supported version for this upgrade guide is **from a 0.7 release to a 0.8 release**.
+The supported version for this upgrade guide is **from a 0.7 release to a 0.8 release**. 
 Build-to-build upgrades are not guaranteed to work.
 This guide is to test upgrades only between the official releases.
 
 For a guide to upgrade previous versions of Rook, please refer to the version of documentation for those releases.
 - [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 - [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
+### Minor Release Upgrades
+The changes in minor releases are scoped to the minimal changes necessary and are expected to be straight forward to upgrade. For upgrades of minor 0.8 releases, see the [minor release upgrade guide](upgrade-minor.md).
```

Comment:
maybe add an explicit example here on a minor release upgrade so people will be able to identify that scenario for themselves more easily, e.g. 0.8.0 to 0.8.1.

also, we may want to consider using the terminology "patch" upgrade, since in semantic versioning it's `MAJOR.MINOR.PATCH`: https://semver.org/

Reply:
agreed, `patch` is the better term here